### PR TITLE
cli: add special code for formatting metrics and settings with -f human

### PR DESCRIPTION
**Pull request by ryanpetrello**
see: https://github.com/ansible/awx/issues/4566

### CODE REVIEW

**Comment by ryanpetrello** [comment]
image

image

**Review by squidboylan — COMMENTED** [review]
This makes sense to me, we might also wanna implement something like this for `awx config`

File: awxkit/awxkit/cli/client.py [hunk 1]
```diff
@@ -159,10 +159,24 @@ def parse_resource(self, skip_deprecated=False):
...
                 response = self.parse_action(resource)
+
+            _filter = self.get_config('filter')
+
+            # human format for metrics and settings is special
+            if self.resource in ('metrics', 'settings') and self.get_config('format') == 'human':
+                response.json = {
+                    'count': len(response.json),
+                    'results': [
+                        {'key': k, 'value': v}
```
Comment (jakemcdermott) [inline awxkit/awxkit/cli/client.py:173]: lgtm

**Review by jakemcdermott — APPROVED** [review]

**Comment by kdelee** [comment]
@ryanpetrello can you roll solution to https://github.com/ansible/awx/issues/4649 into this?

**Comment by ryanpetrello** [comment]
@kdelee this should also address https://github.com/ansible/awx/issues/4649 now
@squidboylan `awx config -f human` and `awx config instances -f human` should work now, too

**Comment by kdelee** [comment]
Like it! Want more of it!
```
awx me -f human
id name 
== ==== 
1       
```

`awx me -f human` could use same treatment

**Comment by ryanpetrello** [comment]
@kdelee try now

**Review by squidboylan — APPROVED** [review]

**Pull request merged**